excel_import_export_demo/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Excel Import/Export/Report Demo",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/server-tools/",
+    "category": "Tools",
+    "depends": ["excel_import_export", "sale_management", "purchase"],
```

Comment:
Add "crm" as dependancy for the tests to correct this runbot errors:

2020-11-02 20:48:29,363 159 WARNING openerp_test odoo.fields: Field report.crm.lead.results with unknown comodel_name 'crm.lead'
2020-11-02 20:48:29,363 159 WARNING openerp_test odoo.fields: Field report.crm.lead.revenue_by_country with unknown comodel_name 'crm.lead'
2020-11-02 20:48:29,363 159 WARNING openerp_test odoo.fields: Field report.crm.lead.revenue_by_team with unknown comodel_name 'crm.lead'